scripts/dockerfiles/Dockerfile.test

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.12-stretch
```

Comment:
Nice, did something similar for metrics in [#413](https://github.com/aws/amazon-vpc-cni-k8s/pull/413/commits/c8faee22a000ca735ae4a69322e11534e753619e#diff-b9d0ccd1235195c35409d2e29dc7c896R1)